src/app/games/siam/tests/SiamMinimax.spec.ts

```diff
@@ -155,7 +154,8 @@ describe('SiamMinimax:', () => {
         ];
         const state: SiamState = new SiamState(board, 0);
         const move: SiamMove = new SiamMove(1, 2, MGPOptional.of(Orthogonal.RIGHT), Orthogonal.RIGHT);
-        const boardValue: number = minimax.getBoardValue(new MGPNode(null, move, state)).value;
+        const boardValue: number = minimax.getBoardValue(
```

Comment:
don't type empty optionals and uniformise that with the others :p ?

Reply:
constantisé, et tu veux uniformiser quelle partie ?

Reply:
uniformisé l'appel à getBoardValue too